xml/virt_support.xml

```diff
@@ -250,26 +265,24 @@
   <para>
    This section lists the support status for various guest operating systems
    virtualized on top of &productname; &productnumber; for &kvm; and &xen;
-   hypervisors. All guest operating
-   systems are supported both fully-virtualized (<quote>FV</quote> in the
-   following table) and paravirtualized (<quote>PV</quote> in the following
-   table) with one exception: Windows, which is only supported fully-virtualized,
-   (but it can use PV drivers <link xlink:href="https://www.suse.com/products/vmdriverpack/"/>).
-   All guest operating systems are supported both in 32-bit and 64-bit flavors,
-   unless stated otherwise.
+   hypervisors.
   </para>
   <itemizedlist>
    <title>Since &sls; 12 SP3, all the following guest operating systems are fully supported (L3):</title>
    <listitem><para>&sls; 11 SP4</para></listitem>
    <listitem><para>&sls; 12 SP3, SP4, SP5</para></listitem>
-   <listitem><para>&sls; 15 SP1, SP2, SP3</para></listitem>
-   <listitem><para>&sled; 15 SP3</para></listitem>
+   <listitem><para>&sls; 15 GA, SP1, SP2, SP3</para></listitem>
    <listitem><para>Windows Server 2008 SP2+, 2008 R2 SP1+, 2012+, 2012 R2+, 2016, 2019</para></listitem>
+   <listitem>
+    <para>
+     Oracle Linux 6,7,8
```

Comment:
@tbazant According to https://jira.suse.com/browse/SLE-17766 description, oracle only plans to support 3rd party kvm vendor, which does not include xen. Right?